appeals/web/src/server/lib/mappers/appeal.mapper.js

```diff
@@ -725,12 +725,12 @@ export async function initialiseAndMapAppealData(
 							{
 								text: 'Yes',
 								value: 'yes',
-								checked: appealDetails.isAffectingNeighbouringSites
+								checked: appealDetails.neighbouringSites?.length > 0
 							},
 							{
 								text: 'No',
 								value: 'no',
-								checked: !appealDetails.isAffectingNeighbouringSites
+								checked: !appealDetails.neighbouringSites?.length
```

Comment:
Should be !appealDetails.neighbouringSites

Reply:
I left that in to add an extra bool in the UI, but can remove the whole row...